docs/user-docs/get_started.md

```diff
@@ -0,0 +1,29 @@
+# Getting Started
+
+- This guide covers just what you need to get trading quickly!
+
+**Before you can start trading on Haveno, you'll need a little Moneroj for a security deposit and fees (0.2 XMR should be enough). Each trader must lock Moneroj in a multisignature escrow until the trade is complete—this is part of what makes trading on Haveno highly secure.**
+
+### 1. [Download and Install Haveno](https://github.com/haveno-dex/haveno/releases)
+
+- It's also a good idea to verify your installer file with our PGP key
+
+If you'd like to build Haveno from source, [here are directions](https://github.com/haveno-dex/Haveno/blob/master/docs/installing.md). Haveno is free/libre open-source software that you can contribute to [(github)](https://github.com/haveno-dex/Haveno).
+
+### 2. Back Up Keys, Write Down Seed
+
+- With Haveno, you're in total control of your funds and your data. This means you retain unparalleled sovereignty, but it also means no one can help you if you lose something important—so it's critical that you do proper backups before using Haveno to trade.
+
+More details at: [data backup and restoration](https://github.com/haveno-dex/Haveno/blob/master/docs/backup_restoration.md).
+
+### 3. Create a Payment Account
+
+- In order to trade monero on Haveno, you've got to set up some way to send or receive other funds. Haveno only handles the monero side of a trade—the other side is handled through fiat payment services (banks, money orders, cash) or altcoin wallets.
+
+Not sure which kind of payment account to set up? There's a full list of [payment methods](https://github.com/haveno-dex/Haveno/blob/master/docs/payment_methods.md) on the wiki.
+
+
+### 4. Do a Trade
+
+Making an offer will usually get you a better price and more control (e.g., setting payment method and deposit percentage), but taking an offer can be more convenient.
+To see both sides of a Haveno trade at the same time, side-by-side: be sure to check out our [trading recommendations](trade-recommendations/payment_methods/README.md) and [linked tutorials made by the community](https://haveno-reto.com/#posts).
```

Comment:
This link needs to be removed, but could be replaced by links directly to the tutorials.